File: 181 excersises/181.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cout << "Ingrese el tamaño de la lista: ";
    cin >> n;
    int *arr = new int[n];
    cout << "Ingresa los " << n << " valores del arreglo:\n";
    for (int i = 0; i < n; i++) {
        cout << "valor " << i + 1 << ": ";
        cin >> arr[i];
    }
    cout << "\nLista ingresada:\n";
    for (int i = 0; i < n; i++) {
        cout << arr[i] << " ";
    }
    cout << endl;
    delete[] arr;

    return 0;
}

File: 181 excersises/181.py
def main():
    n = int(input("Ingresa el tamaño de la lista: "))
    arr = []
    print(f"Ingresa los {n} valores de la lista:")
    for i in range(n):
        elemento = int(input(f"Valor {i + 1}: "))
        arr.append(elemento)
    print("\nLista ingresada:")
    print(" ".join(map(str, arr)))

if __name__ == "__main__":
    main()